### PR TITLE
follow@nightly 0.0.1-nightly.20241021 (new cask)

### PR DESCRIPTION
**Pull request by realsunyz**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).

- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

**Review by bevanjkay — COMMENTED** [review]
Thank you for your contribution @realSunyz 
We only add a versioned cask `@<branch>` when there is a already a stable version offered. In this case, there is not yet at cask for `follow`, and it looks from what I can find on the vendor's website that the suggested version is from the `alpha` branch at the moment. It would be best to first submit a `follow` cask that uses the `alpha` branch, as that is most stable.

**Comment by realSunyz** [comment]
I have opened a new pull request for `follow`, please see #189182.

File: audit_exceptions/github_prerelease_allowlist.json [hunk 1]
```diff
@@ -52,5 +52,6 @@
   "universal-android-debloater": "all",
   "utm@beta": "all",
   "wezterm@nightly": "all",
-  "xit": "all"
+  "xit": "all",
+  "follow@nightly": "all"
```
Comment (p-linnane) [inline audit_exceptions/github_prerelease_allowlist.json:56]: Please add this to the list alphabetically.

File: Casks/f/follow@nightly.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  url "https://github.com/RSSNext/Follow/releases/download/nightly-#{version}/Follow-#{version}-macos-universal.dmg",
+      verified: "github.com/RSSNext/Follow/"
+  name "Follow Nightly"
+  desc "Information browser"
+  homepage "https://follow.is/"
+
+  livecheck do
+    url :url
+    regex(/^nightly-?(\d+(?:\.\d+)+(?:[._-]nightly[._-]?\d+)?)$/i)
```
Comment (khipp) [inline Casks/f/follow@nightly.rb:13]:
```suggestion
    regex(/^nightly[._-]v?(\d+(?:\.\d+)+(?:[._-]nightly[._-]?\d+)?)$/i)
```

**Comment by realSunyz** [comment]
@p-linnane @khipp 
I really appreciate it, I've changed the code as you suggested and updated the version to `0.0.1-nightly.20241021`.

**Review by p-linnane — APPROVED** [review]
Thanks!

**Pull request merged**